### PR TITLE
read_avro

### PR DESCRIPTION
**Pull request by oualib**
- new function read_avro based on the read_json function

### CODE REVIEW

**Comment by oualib** [comment]
@mail4umar can you please check the read_json function and the read_avro function. The other things are not changes, I just moved some positions of functions in the file?

File: verticapy/utilities.py [hunk 1]
```diff
@@ -1415,100 +1415,269 @@ def pjson(path: str, ingest_local: bool = True):
...
+    will create 3 columns: "param.age", "param.name" and "date". 
+    You can rename these columns using the 'new_name' parameter with 
+    the following dictionary:
+    {"param.age": "age", "param.name": "name"}
 insert: bool, optional
-    If set to True, the data is ingested into the input relation.
-    When you set this parameter to True, most of the parameters are 
-    ignored.
+    If set to True, the data will be ingested to the input relation.
+    The AVRO parameters must be the same than the input relation otherwise
```
Comment (mail4umar) [inline verticapy/utilities.py:1462]: "same AS as the input relation"

File: verticapy/utilities.py [hunk 2]
```diff
@@ -1415,100 +1415,269 @@ def pjson(path: str, ingest_local: bool = True):
...
+    When this is true, most of the other parameters will be ignored.
 
 Returns
 -------
 vDataFrame
     The vDataFrame of the relation.
+
+See Also
+--------
+read_json : Ingests a JSON file into the Vertica database.
```
Comment (mail4umar) [inline verticapy/utilities.py:1512]: are we not recording the avro queries and its parameters?

File: verticapy/utilities.py [hunk 3]
```diff
@@ -1517,303 +1686,354 @@ def read_file(
...
     relation types or to customize the data ingestion.
-materialize: bool, optional
-    If set to True, the flex table is materialized into a table.
-    Otherwise, it will remain a flex table. Flex tables simplify the
-    data ingestion but have worse performace compared to regular tables.
+max_files: int, optional
+    (JSON only.) If path is a glob, specifies maximum number of files 
+    in path to inspect. Use this parameter to increase the amount of 
+    data the function considers. This can be beneficial if you suspect 
+    variation among files. Files are chosen arbitrarily from the glob.
```
Comment (mail4umar) [inline verticapy/utilities.py:2009]: maybe add that by default this value is set to 100.

**Review by mail4umar — APPROVED** [review]

**Pull request merged**